neural_clbf/controllers/neural_clbf_controller.py

```diff
@@ -79,6 +80,7 @@ def __init__(
                      effectively trains only a CLF.
             add_nominal: if True, add the nominal V
             normalize_V_nominal: if True, normalize V_nominal so that its average is 1
+            disable_gurobi: if True, gurobi will not be used
```

Comment:
"gurobi will not be used" -> "Gurobi will not be used during evaluation. Default is train with CVXPYLayers, evaluate with Gurobi; setting this to true will evaluate with CVXPYLayers instead (to avoid requiring a Gurobi license)."

Reply:
See [2b05d3e](https://github.com/MIT-REALM/neural_clbf/pull/9/commits/2b05d3e9f2d8a94e18d0b5cb6f31c73a5e3cf4e2)